src/main/java/org/sagebionetworks/bridge/services/ExternalIdService.java

```diff
@@ -104,10 +104,6 @@ public void deleteExternalIdPermanently(Study study, ExternalIdentifier external
         checkNotNull(study);
         checkNotNull(externalId);
         
-        if (study.isExternalIdValidationEnabled()) {
-            throw new BadRequestException("Cannot delete IDs while externalId validation is enabled for this study.");
-        }
```

Comment:
This has been a real pain, so I am switching to making this call and ADMIN-only call (because we only have physical deletes for external IDs). This doesn't remove the account and is pretty safe because you can undo this later if need be by restoring the ID and re-associating it to an account.